### DISCUSSION TITLE
Removing the styles from the Style Gallery/Quick Styles

### DISCUSSION THREAD

**Discussion by techquestions** [dolanmiu/docx#978] [Q&A]
I am adding some styles to my document, as explained in one of the demos.

Is it possible to remove all of the styles from the Style Gallery? What I mean is descibred under [this link](https://support.microsoft.com/en-us/office/add-and-remove-styles-from-the-quick-styles-gallery-21c5b9de-b19e-4575-bc87-cb2b55ece224).

**Comment by dolanmiu** [comment]
At the moment it isnt possible to remove default styles

You could manually override all of them though

What is the use-case of zero styles?
